test/generics.jl

```diff
@@ -0,0 +1,24 @@
+
+# test operators with pd matrix types
+using PDMats
+using Base.Test
+
+# test scalar multiplication 
+print_with_color(:blue, "Testing scalar multiplication\n")
+pm1 = PDMat(eye(3))
+pm2 = PDiagMat(ones(3))
+pm3 = ScalMat(3,1)
+
+pm1a = PDMat(3.*eye(3))
+pm2a = PDiagMat(3.*ones(3))
```

Comment:
Should space out the `.*`, otherwise it's not clear whether it should be dot-broadcast (i.e. `3 .* eye`) or float literal multiplication (i.e. `3. * eye`).